Cargo.toml

```diff
@@ -51,7 +51,8 @@ petgraph = "0.4.13"
 fraction = "0.6"
 sprs = "^0.6"
 ndarray = "=0.12.1"
-rand = "^0.6.5"
+rand = "=0.7"
+rand_xorshift = "=0.2.0"
```

Comment:
TIL: These crates needs to "match" otherwise there will be some trait mismatch when trying to use them, because Rust "versions" each trait based on the package version, so for example if you try to use `rand_xorshift` which relies on `SeedableRng` from `rand` 0.6 but you are using 0.7, that won't work.